hack/installers/checksums/add-helm-checksums.sh

```diff
@@ -3,7 +3,10 @@
 # Usage: ./add-helm-checksums.sh 3.9.4  # use the desired version
 
 set -e
-
 for arch in amd64 arm64 ppc64le s390x; do
   wget "https://get.helm.sh/helm-v$1-linux-$arch.tar.gz.sha256sum"   -O "helm-v$1-linux-$arch.tar.gz.sha256"
 done
+
+for arch in amd64 arm64; do
+  wget "https://get.helm.sh/helm-v$1-darwin-$arch.tar.gz.sha256sum"   -O "helm-v$1-darwin-$arch.tar.gz.sha256"
+done
```

Comment:
Should remove `amd64 arm64` from line 6 as well as they would get 2 binaries downloaded otherwise?

Reply:
I think we can ignore this comment. Did not see linux binary getting downloaded.

Reply:
Yeah, I had to write a separate for loop b/c the other architectures being installed for linux don't exist for darwin.